thk_j1_controller:
  type: thk_ns/MyController
  joint: arm_1_joint
  pid: {p: 100.0, i: 0.01, d: 10.0}

thk_j2_controller:
  type: thk_ns/MyController
  joint: arm_2_joint
  pid: {p: 100.0, i: 0.01, d: 10.0}

thk_j3_controller:
  type: thk_ns/MyController
  joint: arm_3_joint
  pid: {p: 100.0, i: 0.01, d: 10.0}

thk_j4_controller:
  type: thk_ns/MyController
  joint: arm_4_joint
  pid: {p: 100.0, i: 0.01, d: 10.0}

thk_j5_controller:
  type: thk_ns/MyController
  joint: arm_5_joint
  pid: {p: 100.0, i: 0.01, d: 10.0}

thk_j6_controller:
  type: thk_ns/MyController
  joint: arm_6_joint
  pid: {p: 100.0, i: 0.01, d: 10.0}

thk_j7_controller:
  type: thk_ns/MyController
  joint: arm_7_joint
  pid: {p: 100.0, i: 0.01, d: 10.0}


